### ISSUE TITLE
Client timeout value change

### ISSUE THREAD

**Issue by woogerboy21** [Cockatrice/Cockatrice#145]
Currently the client has a hard coded value for the maxTimeout value it expects to receive from the server during it's client->server keep a-lives.  The value is hard coded into the file:
> cockatrice/src/remoteclient.h

It would be nice to move that configuration option out of the header file and into the client configuration settings some were so users can adjust the setting to allow for slower speed connections.

**Comment by woogerboy21** [comment]
@Daenyth So while adding the functionality by reading the setting yet not exposing it I may have come up with a concern on allowing the value to be exposed.  Since the client has no understanding of the settings on the server, if a client adjusts the ping value to a value higher than what the servers operator has set the user time out value, the users client wont check in as often as it would need to and the user would experience constant "server time outs" when the underlying issue is really the user is just contacting the server at to slow of an interval.  After #1313 is worked out, changing the polling interval will be real easy and if we ant to make a UI setting to adjust it thats fine, just as long as there is some type of warning or "hey if you do this , you could cause problems on your end" kind of warning.

**Comment by Daenyth** [comment]
We could add a new field to the server handshake so that the server can inform the client of timeout settings perhaps

**Comment by woogerboy21** [comment]
@Daenyth Should we consider this issue fixed with #1313 or should we keep it open due to the concern mentioned at https://github.com/Cockatrice/Cockatrice/issues/145#issuecomment-126491315 ?

**Comment by Daenyth** [comment]
Can you file a specific ticket about that concern and close this one?

**Comment by woogerboy21** [comment]
Well since the exposed UI setting isn't there and its mentioned that it can be added later (if need be) do we still want to open a new issue on a possibility that could potentially exist assuming the software gets updated to support it?  I don't foresee the UI change occurring anytime soon and not certain anyone is really to interested in doing it.  For me, the client ID communication time has been changed and the value moved to the settings cache which was the primary goal(s).
